curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646ae0eb559a40021a07bae7.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 646ae0eb559a40021a07bae7
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Inside the `div` with the class named `title-container`, You should create a `h1` HTML element with the class name `title` and inside write `Today's Date`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed